src/main/conf/ccm-fetch.cron

```diff
@@ -1,5 +1,4 @@
-# run regularly ccm-fetch. To avoid lost notifications
+# To avoid lost notifications, run ccm-fetch regularly.
 #
-
-0 * * * * root (perl -e 'sleep rand 3600'; /usr/sbin/ccm-fetch) >>/var/log/ccm-fetch.log 2>&1
-
+RANDOM_DELAY=60
+0 * * * * root (date --iso-8601=seconds --utc; /usr/sbin/ccm-fetch) >>/var/log/ccm-fetch.log 2>&1
```

Comment:
cron logs are created with `0640`, and so should the `/var/log/ccm-fetch.log` imho. i'll see if i can let ccm-fetch pick the ccm permissions, so that those would want, can use `world_readable 1` setting.
the problem however is that all logfiles in quattor are created `0644` by default.